package.json

```diff
@@ -68,6 +68,7 @@
     "lib"
   ],
   "dependencies": {
+    "awesome-typescript-loader": "^3.1.3",
```

Comment:
For others not in our slack, I just learned that React Native uses the haste-map when bundling to the App - which means that TypeScript and awesome-typescript-loader are not shipped to Eigen